src/GitVersion.Core.Tests/Core/DynamicRepositoryTests.cs

```diff
@@ -48,7 +48,7 @@ public void Cleanup()
     }
 
     // Note: use same name twice to see if changing commits works on same (cached) repository
-    [NonParallelizable]
+    [NonParallelizable, Ignore("")]
```

Comment:
Please write a comment why this test is ignored.

Reply:
It was an accident. Sorry. I have changed it back.